server/log_rpc_server.go

```diff
@@ -370,6 +370,17 @@ func (t *TrillianLogRPCServer) GetLeavesByRange(ctx context.Context, req *trilli
 	}
 	defer tx.Close()
 
+	if req.ZeroLeavesOk {
+		sth, err := tx.LatestSignedLogRoot(ctx)
+		if err != nil {
+			return nil, err
+		}
+		if req.StartIndex >= sth.TreeSize {
+			// We can't serve any leaves from this range as it starts outside our tree.
+			return &trillian.GetLeavesByRangeResponse{Leaves: []*trillian.LogLeaf{}, TreeSize: sth.TreeSize}, nil
```

Comment:
In this case you can just not set Leaves.

Reply:
It makes the test simpler if it's not nil as nil != []Wibble{}.